zulipterminal/ui_tools/buttons.py

```diff
@@ -118,7 +125,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
 
 class HomeButton(TopButton):
     def __init__(self, *, controller: Any, count: int) -> None:
-        button_text = f"All messages     [{primary_key_for_command('ALL_MESSAGES')}]"
+        button_text = (
+            f"{ALL_MESSAGES_MARKER}"
```

Comment:
These markers can be handled like with other buttons, by passing a `prefix_markup` parameter to `TopButton`.